README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
```

Comment:
Good question... I'm inclined to leave it as-is and, if/as we expand it to include other versions, move their files to subfolders while keeping just one README.